challenges/arrayBinarySearch/array-binary-search.js

```diff
@@ -0,0 +1,25 @@
+const binarySearch = (array, target) => { // trueIndex accounts for the 'halves' of the array that have been trimmed off
+  // error checking
+  if(!Array.isArray(array) || array.length === 0) throw ('binarySearch input must be an array of length 1 or greater');
+  if(isNaN(target)) throw ('binarySearch target value must be a number');
```

Comment:
why does the  target need to be a number?

Reply:
good point, should work for any sorted array so long as default sort algorithm was used.  Examples given were numbers, which is why I went down that restrictive road.